### DISCUSSION TITLE
Next JS 15 - Version mismatch between next-themes and react

### DISCUSSION THREAD

**Discussion by KelumPerera** [vercel/next.js#71929] [Help] [answered]
### Summary

I tried using next 15 and I got this error when installing next-themes.

Appreciate your help!, Thanks! 

`npm i next-themes`

`npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree
npm error
npm error While resolving: finance_dashboard_15@0.1.0
npm error Found: react@19.0.0-rc-69d4b800-20241021
npm error node_modules/react
npm error   react@"19.0.0-rc-69d4b800-20241021" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react@"^16.8 || ^17 || ^18" from next-themes@0.3.0
npm error node_modules/next-themes
npm error   next-themes@"*" from the root project
npm error
npm error Fix the upstream dependency conflict, or retry
npm error this command with --force or --legacy-peer-deps
npm error to accept an incorrect (and potentially broken) dependency resolution.`

**Comment by slythom** [comment, marked as answer]
NextJS 15 uses React 19 RC (...) , making dependencies errors. Try:
`npm i next-themes --force`
or
`npm i next-themes --legacy-peer-deps`

But you may encounter issues.
